README.md

```diff
@@ -178,6 +178,33 @@ settings = {
 See [examples/nvim/init.lua](https://github.com/mrjosh/helm-ls/blob/master/examples/nvim/init.lua) for an
 complete example, which also includes yaml-language-server.
 
+#### Neovim + CoC plugin
+
+Using the [coc.nvim](https://github.com/neoclide/coc.nvim) plugin, you don't need the previous configuration in lua. Coc will handle the configuration instead.
+
+You only need to configure the use of `helm_ls` in the "`langageserver`" section of your `coc-settings.json` file. 
+
+Open Neovim and type the command `:CocConfig` to access the configuration file. Find the `langageserver` section, or create one if necessary, and add this configuration:
+
+
+```json
+"languageserver": {
+  "helm": {
+    "command": "helm_ls",
+    "args": ["serve"],
+    "filetypes": ["yaml", "helm", "helmfile"],
```

Comment:
Im not sure about having yaml here.

How did you setup the filetypes helm and helmfile in your config? With towolf/vim-helm?

Reply:
Yes right.

Maybe I can remove yaml from the list. I will do it today.